jooq/src/main/kotlin/com/terraformation/backend/jooq/Config.kt

```diff
@@ -159,6 +162,7 @@ val ID_WRAPPERS =
                 IdWrapper("InternalTagId", listOf("internal_tags\\.id", ".*\\.internal_tag_id")),
                 IdWrapper("NotificationId", listOf("notifications\\.id", ".*\\.notification_id")),
                 IdWrapper("OrganizationId", listOf("organizations\\.id", ".*\\.organization_id")),
+                IdWrapper("OrganizationTypeId", listOf("organizations\\.type_id")),
```

Comment:
This could also be on the enum table's column list.

Reply:
Oops, I added it to the enum table's list but missed removing it from here.